epoxy-compose/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+  id 'com.android.library'
+  id 'kotlin-android'
+  id 'kotlin-kapt'
+}
+
+android {
+  compileSdkVersion rootProject.COMPOSE_COMPILE_SDK_VERSION
+  buildToolsVersion "30.0.3"
```

Comment:
I think buildToolsVersion can be omitted and the correct version is used by default?